source/jormungandr/jormungandr/scenarios/helper_classes/fallback_durations.py

```diff
@@ -313,3 +333,22 @@ def wait_and_get(self, mode):
 
     def is_empty(self):
         return next((False for _, v in self._value.items() if v.wait_and_get()), True)
+
+    def get_best_fallback_durations(self, main_mode):
+        main_fb = self.wait_and_get(main_mode)
+        res = {}
+        overriding_modes = self._overriding_mode_map.get(main_mode, [])
+        overriding_fbs = [self.wait_and_get(mode) for mode in overriding_modes if self.wait_and_get(mode)]
```

Comment:
I don't understand this line

Reply:
I am getting all fallback_durations of all overriding_modes